enzyme/Enzyme/InstructionDerivatives.td

```diff
@@ -68,6 +70,17 @@ def : CallPattern<(Op $y, $x),
                   (FNeg (FDiv (FMul (DiffeRet<"">), $y), (FAdd (FMul $x, $x), (FMul $y, $y))))
                   ]
                   >;
+
+def : CallPattern<(Op $x),
+                  ["asin", "asinf", "asinl"],
+                  [(FDiv (DiffeRet<"">), (SameSqrt (FSub (ConstantFP<"1.0"> $x), (FMul $x, $x)))  )]
```

Comment:
Can you add support for intrinsics here and use the llvm sqrt intrinsic rather than the std libc function?

Reply:
done